serve.sh

```diff
@@ -1 +1 @@
-bundler exec jekyll serve --future
\ No newline at end of file
+bundler exec jekyll serve --future --incremental
```

Comment:
oh I think it was working but I just turned it on to see what would happen didn't mean to commit it